demo/demo-cse-v2/provider/src/main/resources/application.yml

```diff
@@ -34,6 +34,10 @@ servicecomb:
   kie:
     serverUri: ${CSE_V2_CC}
     customLabel: public
+  handler:
+    chain:
+      Provider:
+        default: governance-provider1
```

Comment:
Add test case to demo-springmvc or other demos, and add assertions to make sure your test case is exactly excuted.